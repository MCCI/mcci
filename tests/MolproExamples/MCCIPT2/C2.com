 MEMORY, 250, M
 R1=1.5
 geometry={C;C,C,R1}
 basis=6-31G
 hf;
{FCI;CORE;DUMP}
