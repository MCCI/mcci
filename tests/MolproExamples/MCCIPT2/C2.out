
 Primary working directories    : /Users/jeremycoe/GreerMCCI/molproV4testPT2/testfiles
 Secondary working directories  : /Users/jeremycoe/GreerMCCI/molproV4testPT2/testfiles
 Wavefunction directory         : /Users/jeremycoe/wfu/
 Main file repository           : /Users/jeremycoe/GreerMCCI/molproV4testPT2/testfiles/

 ARCHNAME  : Darwin/x86_64
 FC        : /usr/local/bin/gfortran
 FCVERSION : 4.5.1
 BLASLIB   : -L/System/Library/Frameworks/Accelerate.framework/Versions/Current/Frameworks/vecLib.framework/Versions/Current -lBLAS
 id        : hw

 Nodes              nprocs
 imac10.eps.hw.ac.uk    1
 ga_uses_ma=false, calling ma_init with nominal heap.
 GA-space will be limited to   8.0 MW (determined by -G option)

 Using customized tuning parameters: mindgm=90; mindgv=120; mindgc=28; mindgr=80; noblas=0; mincuda=1000; minvec=7
 default implementation of scratch files=df  

  MEMORY, 250, M
  R1=1.5
  geometry={C;C,C,R1}
  basis=6-31G
  hf;
 {FCI;CORE;DUMP}

 Variables initialized (664), CPU time= 0.01 sec
 Commands  initialized (462), CPU time= 0.02 sec, 486 directives.
 Default parameters read. Elapsed time= 0.05 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                         Copyright, University College Cardiff Consultants Limited, 2008

                                    Version 2010.1 linked 26 Sep 2013 11:06:42


 **********************************************************************************************************************************
 LABEL *                                                                                 
 Darwin-10.8.0/imac10.eps.hw.ac.uk(i386) 64 bit mpp version                              DATE: 30-Sep-13          TIME: 16:08:22  
 **********************************************************************************************************************************

 Patch level:      13
 **********************************************************************************************************************************
 SETTING R1             =         1.50000000                                  

 Variable memory set to  250000000 words,  buffer space   230000 words

 SETTING BASIS          =    6-31G


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry C      S 6-31G                selected for orbital group  1
 Library entry C      P 6-31G                selected for orbital group  1

1PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Linear
 Symmetry elements: X,Y,Z
 Rotational constants:     133.5620985    133.5620985      0.0000000 GHz  (calculated with average atomic masses)

 Point group  D2h 



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  C       6.00    0.000000000    0.000000000   -0.750000000
   2  C       6.00    0.000000000    0.000000000    0.750000000

 Bond lengths in Bohr (Angstrom)

 1-2  1.500000000
     (0.793765813)

 NUCLEAR CHARGE:                   12
 NUMBER OF PRIMITIVE AOS:          44
 NUMBER OF SYMMETRY AOS:           44
 NUMBER OF CONTRACTIONS:           18   (   5Ag  +   2B3u +   2B2u +   0B1g +   5B1u +   2B2g +   2B3g +   0Au  )
 NUMBER OF CORE ORBITALS:           2   (   1Ag  +   0B3u +   0B2u +   0B1g +   1B1u +   0B2g +   0B3g +   0Au  )
 NUMBER OF VALENCE ORBITALS:        8   (   2Ag  +   1B3u +   1B2u +   0B1g +   2B1u +   1B2g +   1B3g +   0Au  )


 NUCLEAR REPULSION ENERGY   24.00000000


 Eigenvalues of metric

         1 0.207E+00 0.232E+00 0.866E+00 0.119E+01 0.362E+01
         2 0.619E+00 0.264E+01
         3 0.619E+00 0.264E+01
         5 0.608E-03 0.489E-01 0.382E+00 0.965E+00 0.249E+01
         6 0.124E+00 0.614E+00
         7 0.124E+00 0.614E+00


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     0.262 MB (compressed) written to integral file (100.0%)

     Node minimum: 0.262 MB, node maximum: 0.262 MB


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:       2829.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:       2829      RECORD LENGTH: 524288

 Memory used in sort:       0.56 MW

 SORT1 READ        8184. AND WROTE        2829. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.01 SEC
 SORT2 READ        2829. AND WROTE        2829. INTEGRALS IN      1 RECORDS. CPU TIME:     0.01 SEC, REAL TIME:     0.01 SEC

 FILE SIZES:   FILE 1:     2.6 MBYTE,  FILE 4:     4.2 MBYTE,   TOTAL:      6.8 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19        2.57       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *         0.28      0.02
 REAL TIME  *         0.31 SEC
 DISK USED  *         7.52 MB      
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************

1PROGRAM * RHF-SCF (CLOSED SHELL)       Authors: W. Meyer, H.-J. Werner


 NUMBER OF ELECTRONS:       6+    6-
 CONVERGENCE THRESHOLDS:    1.00E-05 (Density)    1.00E-07 (Energy)
 MAX. NUMBER OF ITERATIONS:       60
 INTERPOLATION TYPE:            DIIS
 INTERPOLATION STEPS:              2 (START)      1 (STEP)
 LEVEL SHIFTS:                  0.00 (CLOSED)  0.00 (OPEN) 



 Orbital guess generated from atomic densities. Full valence occupancy:    3   1   1   0   3   1   1   0


 Molecular orbital dump at record        2100.2

 Initial alpha occupancy:   2   1   1   0   2   0   0
 Initial beta  occupancy:   2   1   1   0   2   0   0
 Wave function symmetry:    1

 ITERATION    DDIFF          GRAD             ENERGY        2-EL.EN.            DIPOLE MOMENTS         DIIS
    1      0.000D+00      0.000D+00       -74.45099764     95.768615    0.00000    0.00000    0.00000    0
    2      0.000D+00      0.304D-01       -74.46205624     94.520358    0.00000    0.00000    0.00000    1
    3      0.919D-01      0.642D-02       -74.46278102     94.705395    0.00000    0.00000    0.00000    2
    4      0.181D-01      0.182D-02       -74.46286581     94.665736    0.00000    0.00000    0.00000    3
    5      0.107D-01      0.339D-03       -74.46286848     94.667606    0.00000    0.00000    0.00000    4
    6      0.104D-02      0.557D-04       -74.46286856     94.667403    0.00000    0.00000    0.00000    5
    7      0.338D-03      0.644D-05       -74.46286856     94.667415    0.00000    0.00000    0.00000    5
    8      0.959D-04      0.704D-06       -74.46286856     94.667419    0.00000    0.00000    0.00000    6
    9      0.108D-04      0.165D-07       -74.46286856     94.667419    0.00000    0.00000    0.00000    4
   10      0.197D-07      0.171D-08       -74.46286856     94.667419    0.00000    0.00000    0.00000    0

 Final occupancy:   2   1   1   0   2   0   0

 !RHF STATE 1.1 Energy                -74.462868564322
 Nuclear energy                        24.00000000
 One-electron energy                 -145.79657823
 Two-electron energy                   47.33370966
 Virial quotient                       -0.94198183
 !RHF STATE 1.1 Dipole moment           0.00000000     0.00000000     0.00000000
 Dipole moment /Debye                   0.00000000     0.00000000     0.00000000

 Orbital energies:

         1.1          2.1          3.1          4.1
    -11.409278    -1.156539    -0.047991     0.588660

         1.2          2.2
     -0.638322     0.580725

         1.3          2.3
     -0.638322     0.580725

         1.5          2.5          3.5          4.5
    -11.325794    -0.396326     0.524162     1.172677

         1.6          2.6
      0.345197     1.079296

         1.7          2.7
      0.345197     1.079296


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19        2.57       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              2       3        0.26       700     1000     2100   
                                         GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL        HF       INT
 CPU TIMES  *         0.30      0.03      0.02
 REAL TIME  *         0.33 SEC
 DISK USED  *         7.52 MB      
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************

1PROGRAM * FCI (Full CI)     Author: P.J. Knowles, 1984

 ***  Initialisation ***

 Molecular orbitals read from record     2100.2  Type=RHF/CANONICAL (state 1.1)

 Frozen orbitals:              0 (  0  0  0  0  0  0  0  0)
 Active orbitals:             18 (  5  2  2  0  5  2  2  0)
 Active electrons:            12
 Spin quantum number:          0.0
 Orbital pairs:                    42      20      20       8      33      20      20       8
 Strings:                          -1      -1      -1      -1      -1      -1      -1      -1
                                    0       0       0       0       0       0       0       0
 Determinants wider:                    0           0           0           0           0           0           0           0

 Load integrals                0.4 sec
 Transform integrals           0.4 sec
 Storage for integrals:           5247
 Remaining memory:           249993681

 Core energy:              24.00000000
 Run Hamiltonian processor DUMP    
 Symmetry:                   1
 Trial vector:                  0.00
 Result vector:                 0.00
 Maximum iterations:           90
 Convergence threshold:         0.0000100
 RHS vector:                    0.00
 Hamiltonian shift:             0.0000000
 Output threshold:              0.0500000
 Options:                        0
 Number of roots:             1


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19        2.57       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              2       3        0.26       700     1000     2100   
                                         GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL       FCI        HF       INT
 CPU TIMES  *         0.34      0.02      0.03      0.02
 REAL TIME  *         0.37 SEC
 DISK USED  *         7.52 MB      
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************

        HF-SCF  
    -74.46286856
 **********************************************************************************************************************************
 Variable memory released
